strawberryfields/backends/tfbackend/__init__.py

```diff
@@ -141,6 +141,54 @@
    :members:
 
 """
+import sys
 
 from .backend import TFBackend
 from .ops import def_type as tf_complex_type
+
+try:
+    import tensorflow as tf
+except ImportError:
+    tf_available = False
+else:
+    tf_available = True
+    tf_version = tf.__version__
+
+
+tf_info = """\
+ImportError:
+To use Strawberry Fields with TensorFlow support, version 1.3 of
+TensorFlow is required. This can be installed as follows:
+
+pip install tensorflow==1.3
```

Comment:
Have you thought about having this installed using `extras_require` (i.e. pip install `strawberryfields[tf]`? That seems more standards-based.

Reply:
The only disadvantage would be that tensorflow installation errors would look like they are coming from Strawberry Fields (as it does currently), and I would like to avoid that.

This way, if something goes wrong, it is explicit to the user that the

```python
pip install tensorflow==1.3
```

is failing.